waffle_utils/run.py

```diff
@@ -125,7 +156,9 @@ def _extract_frames(
 ):
     """Extract Frames from a Video File"""
 
-    extract_frames(input_path, output_dir, frame_rate, output_image_extension, verbose)
+    extract_frames(
+        input_path, output_dir, frame_rate, output_image_extension, verbose
+    )
```

Comment:
Has the contents of .pre-commit-config.yaml changed recently? 
It looks like it has been changed even though no new code was added

Reply:
This is my current `.pre-commit-config.yaml`. I have never changed it before.
```yaml
# Define hooks for code formations
# Will be applied on any updated commit files if a user has installed and linked commit hook

default_language_version:
  python: python3.9

exclude: 'docs/'
# Define bot property if installed via https://github.com/marketplace/pre-commit-ci
ci:
  autofix_prs: true
  autoupdate_commit_msg: '[pre-commit.ci] pre-commit suggestions'
  autoupdate_schedule: monthly
  # submodules: true

repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.4.0
    hooks:
      # - id: end-of-file-fixer
      - id: trailing-whitespace
      - id: check-case-conflict
      - id: check-yaml
      - id: check-toml
      - id: pretty-format-json
        args:
          - --autofix
      - id: check-docstring-first

  - repo: https://github.com/asottile/pyupgrade
    rev: v3.3.1
    hooks:
      - id: pyupgrade
        name: Upgrade code
        args: [ --py37-plus ]

  - repo: https://github.com/pycqa/isort
    rev: 5.12.0
    hooks:
      - id: isort
        name: Sort imports
        args: [
          --multi-line=3,
          --split-on-trailing-comma
          ]

  - repo: https://github.com/psf/black
    rev: 22.12.0
    hooks:
    - id: black
      args: [--line-length=79]

  # - repo: https://github.com/PyCQA/flake8
  #   rev: 6.0.0
  #   hooks:
  #     - id: flake8
  #       name: PEP8
  #       args: ["--ignore=E501,E741"]
```